petastorm/spark/spark_dataset_converter.py

```diff
@@ -517,6 +536,25 @@ def wait_for_file(path):
         pool.join()
 
 
+def _check_dataset_file_median_size(url_list):
+    fs, path_list = get_filesystem_and_path_or_paths(url_list)
+
+    # TODO: also check file size for other file system.
+    if isinstance(fs, LocalFileSystem):
+        pool = ThreadPool(64)
+        try:
+            file_size_list = pool.map(os.path.getsize, path_list)
+            mid_index = len(file_size_list) // 2
+            median_size = sorted(file_size_list)[mid_index]
```

Comment:
minor: use median from statistics module

Reply:
python2 don't have `statistics` package so need to install it by pip. Do we need it ?